src/Concerns/CanRender.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Plank\Contentable\Concerns;
+
+use Illuminate\Database\Eloquent\Relations\MorphMany;
+use Plank\Contentable\Contracts\Renderable;
+use Plank\Contentable\Facades\Contentable;
+
+trait CanRender
+{
+    public static function bootCanRender()
+    {
+        static::updated(function (Renderable $module) {
+            foreach ($module->renderable as $content) {
+                Contentable::clearCache($content->contentable->getKey());
+            }
+        });
```

Comment:
If you redefine the `renderable` relation to `content` (see comment below),  and also move the cache clearing logic to the Content Model/Interface (See other suggestion) this should become:

```suggestion
        static::updated(function (Renderable $renderable) {
            $renderable->contentable()?->clearContentCache();
        });
```

Reply:
As mentioned below, since we're getting back a `Collection` instead of a single instance, we need a call to `->each` (or a foreach lookp) here, but otherwise i agree with the suggestions!